app/controllers/spina/conferences/primer_theme/journal/issues_controller.rb

```diff
@@ -13,6 +13,7 @@ class IssuesController < ApplicationController
           def index
             # having multiple journals is not currently allowed anyway
             @issues = Admin::Journal::Issue.sorted_desc
+            @latest_issue = @issues.find { |issue| issue.date <= Time.zone.today }
```

Comment:
The following will get the database to filter the records:
```suggestion
            @latest_issue = @issues.find_by 'date <= ?', Time.zone.today
```

Reply:
Great!